### ISSUE TITLE
Request for an additional command to be added to the desc-- file

### ISSUE THREAD

**Issue by Arvin-O** [abulhassansheikh/O2_P3_DataConsolidation#4]
Hey Abul,

So as per our conversation, here are the details for what I wanted to be added to the automation

I'd like a Y/N column in the NS--File to add a certain line before the Company Information. You can call the column P65 Warning or anything like that.

For example, this is the series features along with the company information:

<div id="features" class="product-key-features"><h3 class="contentheader">Features:</h3><ul><li>100% Aluminum Construction for Long-Lasting Durability and Corrosion Resistance in the Toughest Environments</li><li>The Engineered Aluminum Extruded Main Bar With Aluminum End Caps Fit Tight to the Body Creating the Ultimate in Clean Style</li><li>Patented Drop Down Hoop Step with New Larger Step Area for Ease of Access for Stock or Lifted Vehicles</li><li>Dimpled Step Pad for Secure Footing in Any Condition</li><li>Fully Welded 1-Piece Design for Strength and Ease of Installation</li><li>Backed by a Five-Year Finish / Limited Lifetime Workmanship</li></ul></div><div id="company-info" class="product-description"><p>From nerf bars to light bars to spare tire carriers, N-Fab is the off-road aficionado’s reliable source of great looking and exceptionally durable components to elevate their truck or SUV. The company is regarded for incorporating efficient tubular steel fixtures in their products. They utilize a thicker and heavy gauge steel wall for outstanding resilience. They then implement a superior zinc coating to keep their products from corroding over time. Adding to that, they utilize a high gloss powder coating over the zinc for an awesome appearance.</p>

And what I would like to do is, if we add the p65 warning, it would should this instead:

<div id="features" class="product-key-features"><h3 class="contentheader">Features:</h3><ul><li>100% Aluminum Construction for Long-Lasting Durability and Corrosion Resistance in the Toughest Environments</li><li>The Engineered Aluminum Extruded Main Bar With Aluminum End Caps Fit Tight to the Body Creating the Ultimate in Clean Style</li><li>Patented Drop Down Hoop Step with New Larger Step Area for Ease of Access for Stock or Lifted Vehicles</li><li>Dimpled Step Pad for Secure Footing in Any Condition</li><li>Fully Welded 1-Piece Design for Strength and Ease of Installation</li><li>Backed by a Five-Year Finish / Limited Lifetime Workmanship</li></ul></div><br><p><span class="p65warning"><img src="{{media url='/wysiwyg/warning.jpg'}}" style="max-height:15px;"></span><span style="color:red;font-weight:700">P65 Warning</span>: This product can expose you to chemicals which are known to the State of California to cause cancer or other reproductive harm. Please visit <a href="https://www.p65warnings.ca.gov/"; target="_blank">www.p65warnings.ca.gov</a> for more information.</p><div id="company-info" class="product-description"><p>From nerf bars to light bars to spare tire carriers, N-Fab is the off-road aficionado’s reliable source of great looking and exceptionally durable components to elevate their truck or SUV. The company is regarded for incorporating efficient tubular steel fixtures in their products. They utilize a thicker and heavy gauge steel wall for outstanding resilience. They then implement a superior zinc coating to keep their products from corroding over time. Adding to that, they utilize a high gloss powder coating over the zinc for an awesome appearance.</p>  

<br><p><span class="p65warning"><img src="{{media url='/wysiwyg/warning.jpg'}}" style="max-height:15px;"></span><span style="color:red;font-weight:700">P65 Warning</span>: This product can expose you to chemicals which are known to the State of California to cause cancer or other reproductive harm. Please visit <a href="https://www.p65warnings.ca.gov/"; target="_blank">www.p65warnings.ca.gov</a> for more information.</p> 
 
So, long story short, I would just like to add that line when we put a "Y" in the NS-File.
